Source/Core/DolphinQt2/QtUtils/ListTabWidget.cpp

```diff
@@ -0,0 +1,78 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "DolphinQt2/QtUtils/ListTabWidget.h"
+
+#include <QHBoxLayout>
+#include <QListWidget>
+#include <QScrollBar>
+#include <QStackedWidget>
+
+class FixedWidthAlwaysOneSelectedListWidget : public QListWidget
```

Comment:
Yeah haha it is overly long. Hmm, `CategoryListWidget` confuses me, because categories aren't mentioned anywhere else in this file. What about something like `OverriddenListWidget`? With comments to describe why each function needs to be overridden.